iphone/Classes/TiUIWebView.m

```diff
@@ -443,6 +443,9 @@ - (WKUserScript *)userScriptTitaniumInjectionForAppEvent
     warn: function(message){ \
     window.webkit.messageHandlers._Ti_.postMessage({name:'warn', method: 'log', callback: Ti._JSON({level:'warn', message:message},1)},'*'); \
     }, \
+    log: function(level, message){ \
+    window.webkit.messageHandlers._Ti_.postMessage({name:'level', method: 'log', callback: Ti._JSON({level:'level', message:message},1)},'*'); \
```

Comment:
You're passing in the string literal `'level'` instead of the actual value of the `level` argument.

Also is there any way to add a test for this? Open a Ti.UI.WebView, have it call Ti.API.log('warn', 'message'); and verify it actually occurred somehow? (I'm guessing not, since it goes directly to our APIModule and we can't intercept it)

Reply:
Good catch. Thanks!
I don't think we can write test case for same reason as you have mentioned. Probably we can add event listener in APIModule and fire some event when log api get called. For this we have  to make some modification inside APIModule. I am not sure if it is right way.